docs/onboarding-tech/3-network.md

```diff
@@ -0,0 +1,28 @@
+---
+date: 2024-02-12
+category: document
+author:  Samson Rinaldi / Chris Punt
+order: 3
+---
+
+# Networking: How We're Connected
+
+Welcome to a quick dive into how networking powers connectivity at Jangala! Whether you're tech-savvy or just curious about the technical underpinnings, this guide will provide you with key insights into how our products share connectivity.
+
+## Wi-Fi Connectivity
+
+Our devices broadcast Wi-Fi at two main frequencies: 2.4GHz and 5GHz, which are part of the radio band of the electromagnetic spectrum. The 2.4GHz band offers extensive coverage and greater range, making it ideal for reaching through walls and other barriers. In contrast, the 5GHz band provides faster data speeds, perfect for high-bandwidth activities like streaming and gaming.
```

Comment:
It might be good to include pictures of the spectrum

Reply:
Also the 5GHz band is wider and so can afford larger channels. For the same channel sizes the speeds at 2.4 and 5 are the same

Reply:
done